server/lib/src/server/mod.rs

```diff
@@ -1003,6 +1011,10 @@ impl<'a> QueryServerTransaction<'a> for QueryServerWriteTransaction<'a> {
     fn get_domain_display_name(&self) -> &str {
         &self.d_info.d_display
     }
+
+    fn get_txn_cid(&self) -> &Cid {
```

Comment:
Should be in impl queryServerWrite NOT in impl QueryServerTransaction for QueryServerWrite.